keyboards/keebio/iris/keymaps/mojitas/keymap.c

```diff
@@ -0,0 +1,237 @@
+#include QMK_KEYBOARD_H
+
+//Heavily modified keymap. Some features:
+//Multiple layouts, I use dvorak as main.
+//Nordic(swedish) signs
+//Symbols, numpad, arrows/navigation reachable under the alpas via the layers
+//Mirrored ctl, alt and shift to be able to use both hands when doing commands
+//Gaming layer, qwerty with space on left half.
+
+enum layer_names {
+    _DVORAK,
+    _COLEMAK,
+    _QWERTY,
+    _WORKMAN,
+    _GAMING,
+    _NUMPAD,
+    _LOWER,
+    _RAISE,
+    _ADJUST
+};
+
+enum custom_keycodes {
+  QWERTY = SAFE_RANGE,
+  DVORAK,
+  COLEMAK,
+  WORKMAN,
+  GAMING,
+  NUMPAD,
+};
+
+#define KC_ KC_TRNS
+
+#define KC_Sw2 RALT(KC_2) // Nordic @
```

Comment:
You can `#include "keymap_swedish.h"` to get these Swedish keycodes: https://docs.qmk.fm/#/reference_keymap_extras
https://github.com/qmk/qmk_firmware/blob/master/quantum/keymap_extras/keymap_swedish.h